src/app/[locale]/(houseHold)/[houseHoldId]/(content)/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { db } from "@/drizzle";
+import { HouseHoldTable } from "@/drizzle/schema";
+import { eq } from "drizzle-orm";
+import { notFound } from "next/navigation";
+
+export default async function HouseHoldPage({
+  params,
+}: {
+  params: Promise<{ houseHoldId: string }>;
+}) {
+  const { houseHoldId } = await params;
+  const houseHold = await getHouseHold(houseHoldId);
+
+  if (houseHold == null) notFound();
+  return (
+    <div className=" mx-6">
+      <p>sd</p>
+    </div>
+  );
+}
+
+function getHouseHold(id: string) {
+  const uuidRegex =
```

Comment:
jeśli korzystasz z uuid to pewnie masz gdzieś tam libkę która zwaliduje Ci takie rzeczy: https://www.npmjs.com/package/uuid#uuidvalidatestr